db/migrate/20180801150624_add_test_track_app_and_app_id.rb

```diff
@@ -0,0 +1,13 @@
+class AddTestTrackAppAndAppId < ActiveRecord::Migration[5.0]
+  class App < ActiveRecord::Base; end
+  class IdentifierType < ActiveRecord::Base; end
+  def up
```

Comment:
nit: space between this line and the class definitions above it?

Reply:
👍